commands/command_edit.cpp

```diff
@@ -1061,6 +1061,7 @@ void AddCylinder::undo()
 		if (d->cylinders.nr <= 0)
 			continue;
 		remove_cylinder(d, d->cylinders.nr - 1);
+		update_cylinder_related_info(d);
```

Comment:
still can't comment on commit messages... so this is on the commit message for "undo: update cylinder related info when editing cylinder data" - you have the word "Gor" in there, which I'm not familiar with :-)